packages/@coorpacademy-components/src/atom/loader/index.native.tsx

```diff
@@ -0,0 +1,215 @@
+import {useAnimateProp} from '@coorpacademy/react-native-animation';
+import React, {useEffect, useRef, useState} from 'react';
+import {Animated, StyleSheet, Easing, ViewStyle, ColorValue} from 'react-native';
+import {useTemplateContext} from '../../template/app-review/template-context';
+import {Theme} from '../../variables/theme.native';
+
+type StyleSheetType = {
+  container: ViewStyle;
+  dot: ViewStyle;
+  blue: ViewStyle;
+  red: ViewStyle;
+  green: ViewStyle;
+  yellow: ViewStyle;
+};
+
+const createStyleSheet = (theme: Theme): StyleSheetType =>
+  StyleSheet.create({
+    container: {
+      alignItems: 'center',
+      justifyContent: 'center'
+    },
+    dot: {
+      position: 'absolute'
+    },
+    blue: {
+      backgroundColor: theme.colors.cta
+    },
+    red: {
+      backgroundColor: theme.colors.negative
+    },
+    green: {
+      backgroundColor: theme.colors.positive
+    },
+    yellow: {
+      backgroundColor: theme.colors.battle
+    }
+  });
+
+export type Props = {
+  color?: ColorValue;
+  height?: number;
+  readyToHide: boolean;
+};
+
+const CYCLE_DURATION = 3000;
+const MIN_TIME = 1000;
+
+const Loader = (props: Props) => {
```

Comment:
Where do you call this component? Shouldn't the template component src/template/app-review/index.native.tsx (l. 19) be modified?